Sources/Stores/Store.swift

```diff
@@ -85,8 +86,10 @@ public extension Store {
             }
 
             do {
-                let value = try resource.parser(data)
-                completion(value, nil, true)
+                try self.metrics?.measure(with: "Parse of \(T.self) with size: \(data.endIndex)") {
```

Comment:
we could make the key configurable by injecting a `parseMetricKey(typeName: String, dataSize: Int)`
or `parseMetricKey(type: Any.Type, dataSize: Int)` which would default to `"Parse of \(type) with size: \(dataSize)"`

Reply:
Additionally, if `metrics` is `nil` the parse will never be made 💥 🙈 😆